src/activities/hoc2019/PondCreator.jsx

```diff
@@ -81,6 +70,9 @@ export default class PondCreator extends React.Component {
   };
 
   render() {
+    if (!this.state.initialized) {
```

Comment:
does `initialized` ever get set to `true`?

Reply:
nvm, the answer was right in front of me lol

Reply:
Yeah I don't know if this is strictly necessary but I want to avoid trying to use the trainer before it's initialized.